### PR TITLE
N28: Veloren

### PR DESCRIPTION
**Pull request by AngelOnFira**
Part of #852 :100:

### CODE REVIEW

File: content/news/028/index.md [hunk 1]
```diff
@@ -135,6 +135,46 @@ They also worked on a [tutorial][bitgun-tutorial].
...
+![Travelling merchant](veloren.jpg)
+_A merchant paused in their travels_
+
+[Veloren][veloren] is an open world, open-source voxel RPG inspired by Dwarf
+Fortress and Cube World.
+
+In November, Veloren started a new initiative to help new developers learn more
+about the code base; the Veloren Reading Club. You can watch the
+[first][veloren-reading-club-1] and [second][veloren-reading-club-2] sessions
+now. Hats were merged, and  a major rewrite of the server-hosting section of the
```
Comment (17cupsofcoffee) [inline content/news/028/index.md:149]:
```suggestion
now. Hats were merged, and a major rewrite of the server-hosting section of the
```

File: content/news/028/index.md [hunk 2]
```diff
@@ -135,6 +135,46 @@ They also worked on a [tutorial][bitgun-tutorial].
...
+book happened. New textures were made for item drops. Some concept art was
+created for what massive cities could looks like. Dynamic weather was added, and
+you can [watch that in action][veloren-rain-storm].
+
+Experience sharing went through a large overhaul to evenly distribute EXP gained
+across groups. The difficulty of dungeons was adjusted to be more balanced for
+new players. Persistence was added to skills, and measures were put in place to
+help with future migrations to new skill trees. Site2 works was done to help
```
Comment (17cupsofcoffee) [inline content/news/028/index.md:157]: Would it be possible to clarify what `site2` actually is? I thought this was a typo at first, but it sounds like it's part of the world generation systems?

**Review by 17cupsofcoffee — APPROVED** [review]
Couple of minor comments, but looks alright.

File: content/news/028/index.md (hunk 1 above)
Comment (AngelOnFira) [reply content/news/028/index.md:149]: I told myself I would go back to this while I was racing through the content, but I suppose I forgot :smile: thanks for the catch!

File: content/news/028/index.md (hunk 2 above)
Comment (AngelOnFira) [reply content/news/028/index.md:157]: Ya, I didn't want to dive too deep into topics, but I can definitely swap this out for a short sentence :)

**Review by 17cupsofcoffee — APPROVED** [review]
🚀

**Pull request merged**